common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -5615,7 +5615,12 @@ public static enum ConfVars {
 
     HIVE_ADDITIONAL_CONFIG_FILES("hive.additional.config.files", "",
             "The names of additional config files, such as ldap-site.xml," +
-                    "tez-site.xml, etc in comma separated list.");
+                    "tez-site.xml, etc in comma separated list."),
+
+    REPL_FILTER_TRANSACTIONS("hive.repl.filter.transactions", false,
+            "Enable transaction event filtering to save dump space.\n" +
+                    "When true, transactions are implicitly opened during REPL DUMP.\n" +
```

Comment:
Not sure about the `RESTORE` - we need to use the same names as we use other filters.

```
"Enable transaction event filtering to save dump space.\n" +
        "When true, opening transactions do not generate events, and transactions on the target " +
       "server are implicitly opened during REPL RESTORE.\n" +
       "The default setting is false");
```

We want this config to be around the other `REPL` related configs, not at the end of the list.

Reply:
I will move.